web/views/password_view.ex

```diff
@@ -0,0 +1,10 @@
+defmodule CodeCorps.PasswordView do
+  use CodeCorps.Web, :view
+
+  def render("show.json", %{email: email}) do
+    %{
+      email: email
+    }
+  end
```

Comment:
We could use a view test here and definitely assert that the token is never being rendered.